Bio/Phenotype/pm_fitting.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2014 by Marco Galardini.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""This module provides functions to perform sigmoid functions fitting to
```

Comment:
Can you give this docstring a one line module summary? See PEP257 https://www.python.org/dev/peps/pep-0257/